imp_developer.py

```diff
@@ -263,6 +303,20 @@ def show_settings_value_in_status(self, property_name, status_key, formatted_str
         else:
             log_debug("Property \"" + property_name + "\" is not found in the settings")
 
+class HttpHeaders():
```

Comment:
Extra empty line is needed before the class definition.

Reply:
fixed for all classes